sdk/servicebus/service-bus/tests.yml

```diff
@@ -27,6 +27,5 @@ jobs:
         AAD_CLIENT_ID: $(aad-azure-sdk-test-client-id)
         AAD_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
         AAD_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
-        AZURE_SUBSCRIPTION_ID: $(AZURE_SUBSCRIPTION_ID)
         SERVICEBUS_CONNECTION_STRING: $(SERVICEBUS_CONNECTION_STRING)
-        RESOURCE_GROUP: $(RESOURCE_GROUP)
\ No newline at end of file
+        SERVICEBUS_CONNECTION_STRING_BROWSER: $(SERVICEBUS_CONNECTION_STRING_BROWSER)
```

Comment:
The arm template work here ensures that each test run (node or browser) gets its own service bus namespace. So, can we not re-use the same `SERVICEBUS_CONNECTION_STRING` env variable for browser? It will require us to update the 
`envVarUtils.ts` file to not expect the _BROWSER suffix in the env var name for the connection string.

Reply:
But to be able to run them in parallel, they need to point to two different service bus namespaces right? Having one entry would be attempting to create with same name

Reply:
The 2 runs in parallel will be in 2 different processes.
The env var name would be same.
The value for the env var in the 2 processes will be different thanks to the arm template which is expected to generate a uniquely named service bus namespace each time

Reply:
cc @sadasant if my understanding of the expectation of the arm template is wrong above

Reply:
Yes that makes sense - just figured the use of `baseName` parameter - updated.